### PR TITLE
CKAN 210 rebase main

### PR DESCRIPTION
**Pull request by FuhuXia**
This is what we got after `git rebase main -Xtheirs` to bring latest `main` changes into `ckan-210` but keep ckan-210 in any requirements conflicts.

### CODE REVIEW

File: .github/workflows/ckan.yml [hunk 1]
```diff
@@ -10,7 +10,6 @@ on:
         type: choice
         options:
           - development
-          - staging
```
Comment (btylerburton) [inline .github/workflows/ckan.yml:13]: did we remove staging just to speed up runs?

File: .profile [hunk 2]
```diff
@@ -157,7 +159,7 @@ if [[ $MIGRATE_DB = 'True' ]]; then
   # Run migrations
   ckan db upgrade
   ckan harvester initdb
-  ckan archiver init
-  ckan report initdb
-  ckan qa init
+  # ckan archiver init
```
Comment (btylerburton) [inline .profile:162]: where do we init archiver now?

File: ckan/requirements.in [hunk 3]
```diff
@@ -8,21 +8,26 @@ git+https://github.com/GSA/ckanext-saml2auth.git@create_user_via_saml#egg=ckanex
...
-ckanext-googleanalyticsbasic
-ckanext-metrics-dashboard
+# ckanext-datagovcatalog>=0.0.3
+# ckanext-datagovtheme>=0.1.22
+# ckanext-datajson>=0.1.13
+ckanext-envvars>=0.0.3
+# ckanext-geodatagov>=0.1.28
+# ckanext-googleanalyticsbasic
+# ckanext-metrics-dashboard
+-e git+https://github.com/gsa/ckanext-geodatagov.git@ckan-2.10#egg=ckanext-geodatagov
```
Comment (btylerburton) [inline ckan/requirements.in:18]: these should revert to new 2.10 published versions

**Review by btylerburton — APPROVED** [review]
just a few questions, but overall looks good to merge into feature branch

File: .github/workflows/ckan.yml (hunk 1 above)
Comment (FuhuXia) [reply .github/workflows/ckan.yml:13]: nah. just to stop anyone from manually running a GH action when the DB is supposed to be frozen during upgrade.

File: .profile (hunk 2 above)
Comment (FuhuXia) [reply .profile:162]: This commit is already in ckan-210. https://github.com/GSA/catalog.data.gov/commit/f7c28fea7b5042a69fb1ddc06cd57ac78633c3be
I am surprised it shows up here in the change list when I do rebasing main.

File: ckan/requirements.in (hunk 3 above)
Comment (FuhuXia) [reply ckan/requirements.in:18]: Yes it should. We will correct it once `ckan-210` is merged into the `main`. again i am surprized the existing changes in `ckan-210` shows up here in a `git rebase main` result.

**Pull request merged**